modules/gperf/3.1/patches/gperf.patch

```diff
@@ -0,0 +1,190 @@
+From 6c073922a7b04fe520d8350ab4da03d6c8207703 Mon Sep 17 00:00:00 2001
+From: Dmitry Mikhin <dmikhin@webmonitorx.ru>
```

Comment:
nit: why do we need this patch? Can you upstream it?

Reply:
Since cc_library only understands a hardcoded set of file extensions correctly, it doesn't handle *.icc files, so they were all renamed to *.inc.
https://bazel.build/versions/7.0.0/reference/be/c-cpp#cc_library.srcs
![image](https://github.com/user-attachments/assets/4503dc07-afe6-4d9c-85d4-3f0b6f07c9cb)

I'm not a gperf maintainer and I suspect they might reject this patch because it's bazel-specific behavior.
I prefer to leave this patch in BCR.

Reply:
Oh, right, this might need a change in the Bazel cc rules.